src/pcapi/routes/offers.py

```diff
@@ -76,24 +66,10 @@ def get_offer(offer_id: str) -> GetOfferResponseModel:
 @login_or_api_key_required
 @spectree_serialize(response_model=OfferResponseIdModel, on_success_status=201)  # type: ignore
 def post_offer(body: PostOfferBodyModel) -> OfferResponseIdModel:
-    venue = load_or_raise_error(VenueSQLEntity, body.venue_id)
-    ensure_current_user_has_rights(RightsType.editor, venue.managingOffererId)
-
-    if body.product_id:
-        offer = initialize_offer_from_product_id(body.product_id)
-    else:
-        offer = fill_offer_with_new_data(request.json, current_user)
-        offer.product.owningOfferer = venue.managingOfferer
-
-    offer.venue = venue
-    offer.bookingEmail = body.booking_email
-    repository.save(offer)
-    send_offer_creation_notification_to_administration(
-        offer, current_user, PRO_URL, send_raw_email
+    return OfferResponseIdModel.from_orm(
+        post_offer_api(offer_data=body, current_user=current_user)
     )
```

Comment:
Ça me semblerait plus lisible avec une variable temporaire. Et puisque de toute façon on a(ura) la limite de longueur de ligne de Black, au final on gagne même une ligne. :)

```suggestion
    offer = post_offer_api(offer_data=body, current_user=current_user)
    return OfferResponseIdModel.from_orm(offer)
```

Reply:
J'ai hésité un peu là-dessus, j'ai fait ça au dernier moment donc ok pour le revert